integrations/Drip/browser.js

```diff
@@ -0,0 +1,160 @@
+import get from "get-value";
+import logger from "../../utils/logUtil";
+import { removeUndefinedAndNullValues } from "../utils/commonUtils";
+
+class Drip {
+  constructor(config) {
+    this.accountId = config.accountId;
+    this.campaignId = config.campaignId;
+    this.name = "DRIP";
+  }
+
+  init() {
+    logger.debug("===In init Drip===");
+
+    var _dcq = _dcq || [];
```

Comment:
we should do this.
`window.__dcq = _dcq || [];`
`window._dcs = _dcs || {};`
 and use window._dcq and window._dcs everywhere